### ISSUE TITLE
Feature - Support for StringIO

### ISSUE THREAD

**Issue by acatighera** [koshigoe/brick_ftp#110]
# This is a Feature Proposal

## Description

Support for `StringIO` uploads would be nice. There already is support for `IO` so should just be a matter of changing the `if IO` check to `if IO || StringIO`.

**Comment by acatighera** [comment]
```
diff --git a/lib/brick_ftp/http_client.rb b/lib/brick_ftp/http_client.rb
index 78ac2e9..48a7690 100644
--- a/lib/brick_ftp/http_client.rb
+++ b/lib/brick_ftp/http_client.rb
@@ -99,7 +99,7 @@ module BrickFTP
       case
       when params.is_a?(Hash)
         req.body = params.to_json unless params.empty?
-      when params.is_a?(IO)
+      when params.is_a?(IO) || params.is_a?(StringIO)
         req.body_stream = params
         req['Content-Length'] = params.size
       end
```

**Comment by koshigoe** [comment]
I updated by #114 and released [v0.8.3](https://rubygems.org/gems/brick_ftp/versions/0.8.3).

_NOTE: I will release the v1.0.0 (includes breaking changes) in a few days._
